eng/tools/dependency-testing/templates/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist"
+    "outDir": "dist-esm"
```

Comment:
Is this necessary?

Reply:
I just want to be consistent with what our libraries are doing, so that the integration-test script is consistent with what we already have in the pakage.json

Reply:
Ok that makes sense! Thanks!